website/addons/wiki/views.py

```diff
@@ -356,6 +363,44 @@ def project_wiki_edit_post(auth, wname, **kwargs):
         ret = {'status': 'success'}
     return ret, http.FOUND, None, redirect_url
 
+@must_be_valid_project  # injects node or project
+@must_have_permission('admin')
+@must_not_be_registration
+@must_have_addon('wiki', 'node')
+def edit_wiki_permissions(**kwargs):
+    node = kwargs['node'] or kwargs['project']
+    wiki_settings = node.get_addon('wiki')
+    permissions = kwargs.get('permissions')
+    auth = kwargs['auth']
+
+    if permissions is None:
```

Comment:
Can be simplified:

```python
if not permissions and wiki_settings:
    raise HTTPError(http.BAD_REQUEST)
```

Note that this isn't *quite* the same as using `is None` - but those those things should evaluate to `True` by default, and I would merely test their truthiness.